evap/contributor/tests/test_forms.py

```diff
@@ -147,7 +147,7 @@ def test_editors_cannot_delete_responsibles(self):
             "contributions-0-id": "{}".format(contribution.pk),
             "contributions-0-contributor": "{}".format(user.pk),
             "contributions-0-does_not_contribute": "on",
-            "contributions-0-responsibility": "RESPONSBILE",
+            "contributions-0-responsibility": "RESPONSIBLE",
```

Comment:
the changes in the second commit are completely obsoleted by the fourth commit as far as i can see. could you remove it?

Reply:
well, they do different things. the commit message would be something like "lots of cleanups and somewhere in there are also behavioral fixes", which to me would look like that should be two separate commits. what do you think?

Reply:
yeah, fixing stuff that's going to be removed doesn't make much sense. but, here it got refactored, not removed. if you are moving a method somewhere else, i would totally expect you to not include some subtle behavioral changes in that move just because "it got removed from the original place, so no need to fix it there".

I prefer having behavioral changes and refactorings separated. in case something breaks, and bisecting shows it broke during a commit that says "rename foo to bar" and touches a few hundred lines, and for some reason there is some behavioral change in there because the rename would have made a separate behavior-change-commit obsolete, that would suck. likewise, if during review some commit says "rename foo to bar" i would like to trust the author that the commit does exactly that, i don't want to actually closely inspect every line because the author might have put behavioral changes in there as well.

Reply:
i don't see it as a behavioral change in this case because you didn't rename anything, but fixed a typo that can't happen after the refactoring anymore. i see your point though and i could imagine a situation where we want to revert some refactoring which of course shouldn't add wrong code.
maybe we should then pay more attention to such strict separations in the future? because it doesn't feel like we thoroughly did that so far.

i still don't really like the second commit but guess it's okay to do it like that.